build.gradle.kts

```diff
@@ -334,6 +335,8 @@ allprojects {
                     trimTrailingWhitespace()
                     indentWithSpaces(4)
                     endWithNewline()
+                    paddedCell() // Avoid cyclic ambiguities
+                    greclipse().configFile(grEclipseConfFile)
```

Comment:
Is `paddedCell()` mandatory?
The documentation of Spotless says it should be used as a last resort only.

If we can't go without it I would refrain from adding `greclipse` altogether.

Reply:
It doesn't seem to work with or without it; so I can't say if it's required or not.